api/cache.go

```diff
@@ -36,3 +37,37 @@ func cachePurge(c echo.Context) error {
 	}
 	return c.String(http.StatusOK, fmt.Sprintf("Object purged on %d servers", count))
 }
+
+func cachePurgeBulk(c echo.Context) error {
+	if c.Request().ContentLength == 0 {
+		return echo.NewHTTPError(http.StatusBadRequest, "Request body can't be empty")
+	}
+
+	name := c.Param("instance")
+	if len(name) == 0 {
+		return c.String(http.StatusBadRequest, "instance is required")
+	}
+	manager, err := getManager(c)
+	if err != nil {
+		return err
+	}
+
+	var argsList []rpaas.PurgeCacheArgs
+	if err = json.NewDecoder(c.Request().Body).Decode(&argsList); err != nil {
+		return c.String(http.StatusBadRequest, err.Error())
+	}
+
+	var results []rpaas.PurgeCacheBulkResult
+	for _, args := range argsList {
+		var r rpaas.PurgeCacheBulkResult
+		count, err := manager.PurgeCache(c.Request().Context(), name, args)
+		if err != nil {
+			r = rpaas.PurgeCacheBulkResult{Path: args.Path, Error: err.Error()}
+		} else {
+			r = rpaas.PurgeCacheBulkResult{Path: args.Path, InstancesPurged: count}
+		}
+		results = append(results, r)
+	}
+
+	return c.JSON(http.StatusOK, results)
```

Comment:
@pedrokiefer could you change to emit 500 when some error occur ?

Reply:
I can, but does it really makes sense? Say I'm asking to purge 10 URLs, only one fails, should it really be 500?

Reply:
it is important to we detect that have some problem with that route, we could use metrics to achieve that. @pedrokiefer